sdk/storage/azure-storage-file-share/src/test/java/com/azure/storage/file/share/FileServiceAPITests.groovy

```diff
@@ -235,4 +252,60 @@ class FileServiceAPITests extends APISpec {
         INVALID_ALLOWED_ORIGIN | 400        | ShareErrorCode.INVALID_XML_DOCUMENT
         INVALID_ALLOWED_METHOD | 400        | ShareErrorCode.INVALID_XML_NODE_VALUE
     }
+
+    def "Restore share min"() {
+        given:
+        def shareClient = primaryFileServiceClient.getShareClient(generateShareName())
+        shareClient.create()
+        def fileName = generatePathName()
+        shareClient.getFileClient(fileName).create(2)
+        shareClient.delete()
+        if (TestMode.PLAYBACK != getTestMode()) {
+            Thread.sleep(30000)
+        }
```

Comment:
I believe there is a utility method in APISpec that does this.

Reply:
I've seen that in blobs but not in shares. I'll double check and add if missing.